File: components/Issue.js
import React from 'react'

function Issue({commitdata}) {
  return (
    <div>
             <a href="#" className="list-group-item list-group-item-action">
          <div className="d-flex w-100 justify-content-between">
              <h5 className="mb-1">{commitdata.title}</h5>
              {/* <small className="text-muted">4 days ago</small> */}
            </div>
            <p className="mb-1">status: {commitdata.state}</p>
            <div><small className="text-muted">number of comments: {commitdata.comments}</small></div>
            <small className="text-muted">submitted by {'"'}{commitdata.user}{'"'}</small>
          </a>
    </div>
  )
}

export default Issue

File: src/main/java/boyd/api/model/Issue.java
package boyd.api.model;

import lombok.Data;

@Data
public class Issue {

    private String title;
    private String user;
    private String state;
    private int comments;
    private String description;

    public Issue() {}

    public Issue(String title, String user, String state, int comments, String description) {
        this.title = title;
        this.user = user;
        this.state = state;
        this.comments = comments;
        this.description = description;
    }

}
